handlers/messages.go

```diff
@@ -40,6 +40,15 @@ func MessageHandler(session *discordgo.Session, msg *discordgo.MessageCreate) {
 		case "echo": // echo the message that was sent
 			log.Println("msghandler: caught echo command")
 			echo(session, msg, args)
+		
+		case "glizzy": // glizzy
+			log.Println("msghandler: caught glizzy command")
+			// TODO: get these emojis not in a hard-coded way
+			_, err := session.ChannelMessageSend(msg.ChannelID, "<a:glizzyR:991176701063221338>" + strings.Join(args, " ") + "<a:glizzyL:991176582402150531>" )
```

Comment:
The command states <emote> as the arg but that isn't enforced. you could write whatever you want inside the command not just one emote. That might be fine but If the command should only allow one emoji it should be enforced.